include/linux/pps_kernel.h

```diff
@@ -67,6 +67,7 @@ struct pps_device {
  */
 
 extern const struct attribute_group *pps_groups[];
+extern int pps_max_sources;
```

Comment:
Thoughts on `max_sources` instead of `pps_max_sources` since it's a sysfs parameter now?

Reply:
Can you clarify what you mean by `since it's a sysfs parameter now`?
Since this is a global variable, I prefer `pps_max_sources` for more specificity.

Reply:
My thought was that it will only be referred to with the context of pps. Previously, code might have `PPS_MAX_SOURCES` in it with no other context. Making it a module parameter means that `pps_core` will typically be part of the calling convention indicating it's part of pps. That being said, I think the more specificity is good since it's used in pps without context so I'll leave it as-is.